CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 ## dbt-teradata 1.0.0a
 
 ### Features
+* Merge incremental strategy is now supported in dbt-teradata
 
 ### Fixes
-
+* Fixed the bug (https://github.com/Teradata/dbt-teradata/issues/77)
```

Comment:
Can you please describe the issue you fixed, instead of an issue number? like line number 11

Reply:
Done